dev-tools/prepare_release_candidate.py

```diff
@@ -60,10 +60,10 @@
 
 The same goes for the x-plugins:
 
-    bin/elasticsearch-plugin -Des.plugins.staging=true install license
-    bin/elasticsearch-plugin -Des.plugins.staging=true install marvel-agent
-    bin/elasticsearch-plugin -Des.plugins.staging=true install shield
-    bin/elasticsearch-plugin -Des.plugins.staging=true install watcher
+    ES_JAVA_OPTS="-Des.plugins.staging=true" bin/elasticsearch-plugin install license
```

Comment:
This are actually just plain wrong now since we no longer have individual plugins, but unrelated to this change (I'll fix in a followup).